tests/util/fake_statsd_server.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Is Python3 required?

```suggestion
#!/usr/bin/env python
```

Reply:
Yeah that should work too - it's not required.